cmd/renterd/config.go

```diff
@@ -68,6 +68,9 @@ func defaultConfig() config.Config {
 				MetricsDatabase: "renterd_metrics",
 			},
 		},
+		Explorer: config.ExplorerData{
+			URL: "https://api.siascan.com",
```

Comment:
Might want to f/u with a refactor of our config to be consistent about `Enabled` and `Disable`. 
It's a bit confusing that let's say worker, s3 and explorer are all on by default but the worker and s3 are disabled by setting enabled to false and the explorer is disabled by setting disable to true.

Reply:
Yeah I suggested that somewhere. I'll open an issue for it.